RecoTauTag/Configuration/python/boostedHPSPFTaus_cff.py

```diff
@@ -34,8 +34,10 @@
 
 from Configuration.Eras.Modifier_pp_on_XeXe_2017_cff import pp_on_XeXe_2017
 from Configuration.Eras.Modifier_pp_on_AA_2018_cff import pp_on_AA_2018
-for e in [pp_on_XeXe_2017, pp_on_AA_2018]:
+from Configuration.Eras.Modifier_pp_on_PbPb_run3_cff import pp_on_PbPb_run3
+for e in [pp_on_XeXe_2017, pp_on_AA_2018, pp_on_PbPb_run3]:
     e.toModify(ca8PFJetsCHSprunedForBoostedTaus, inputEtMin = 999999.0)
+    e.toModify(ca8PFJetsCHSprunedForBoostedTaus, src = cms.InputTag("particleFlow"))
```

Comment:
```suggestion
    e.toModify(ca8PFJetsCHSprunedForBoostedTaus, inputEtMin = 999999.0, src = "particleFlow")
```
no need to repeat the target